tests/to_device_test.go

```diff
@@ -73,7 +73,7 @@ func TestUnprocessedToDeviceMessagesArentLostOnRestart(t *testing.T) {
 	ForEachClientType(t, func(t *testing.T, clientType api.ClientType) {
 		// prepare for the test: register all 3 clients and create the room
 		tc := CreateTestContext(t, clientType, clientType)
-		roomID := tc.CreateNewEncryptedRoom(t, tc.Alice, "private_chat", []string{tc.Bob.UserID})
+		roomID := tc.CreateNewEncryptedRoom(t, tc.Alice, EncRoomOptions.Invite([]string{tc.Bob.UserID}))
```

Comment:
Missing private chat.

Reply:
It absolutely cannot be left blank when we hit `/createRoom` as Synapse/Dendrite disagree on what the default should be (Dendrite/Conduit [follow the spec](https://spec.matrix.org/v1.9/client-server-api/#post_matrixclientv3createroom) and end up making private by default, but Synapse does public, which can lead to inconsistently failing tests depending on the server). I'm happy for _our code_ to default to `private_chat` though.

Reply:
It's our code that is defaulting to private_chat.